### ISSUE TITLE
AutoNumber doesn't allow persian or arabic number in input box via android 

### ISSUE THREAD

**Issue by sonsy-soft** [autoNumeric/autoNumeric#668]
when in android i want to insert a persian digit in input box that initialized by autoNumber, it rejected the key and prevent to change the value in the input box.

how can i fix it in my code ?

**Comment by AlexandreBonneau** [comment]
Do you have to compose the numbers, or can you input it directly?

**Comment by sonsy-soft** [comment]
yes some library like pesianjs convert persian or araic digit to english we can use it 
https://github.com/usablica/persian.js

**Comment by AlexandreBonneau** [comment]
I'm not sure to follow. Are you using persian.js in combination with AutoNumeric ?
Could you please describle _precisely_ what you are doing to reproduce the bug ?
Would you have a screencast if you feel reproducing it will be hard for non-arabic/persian speakers ?

For instance, when you enter `۳۴۵` in the autoNumeric field, do you get `345`?
For instance, when you enter `٣٤٥` in the autoNumeric field, do you get `345`?

**Comment by sonsy-soft** [comment]
dear Alexandre
I don't get the 345 , because autoNumber reject the arabic/persian number on keypress.
this problem was appeared in smartphone.

**Comment by AlexandreBonneau** [comment]
Which browser are you using under Android?
Do you see any errors in the console when trying to enter arabic numbers in the AutoNumeric element?

**Comment by sonsy-soft** [comment]
i'm using chrome
i use imputmask with my input form that it dosnt have the error

**Comment by JeroenKocyigit** [comment]
Hi @AlexandreBonneau, I am currently experiencing a similar issue, however only on iOS, I don't use any external library.

When you're using an Arabic keyboard, you're unable to enter any values. I looked trough the source code and I believe either the `keydown` or `keyup` event is blocking it the input because the string key cant be converted to a number.

If you need more info let me know, its a requirement for my pwa to enter Eastern Arabic numbers. So I will gladly help in order to figure this out.

![IMG_E0003](https://user-images.githubusercontent.com/19681724/125794609-26b84161-15a5-43a8-8423-03d04584e183.JPG)
